websites/S/SOAP2DAY/dist/metadata.json

```
@@ -12,16 +12,13 @@
 		"soap2day.to",
 		"soap2day.ac",
 		"soap2day.sh",
-		"soap2day.cx",
-		"soap2day.rs",
+		"soap2day.mx",
 		"s2dfree.to",
 		"s2dfree.cc",
 		"s2dfree.de",
-		"s2dfree.is",
-		"s2dfree.nl",
-		"secretlink.xyz"
+		"s2dfree.is"
 	],
-	"version": "1.1.7",
+	"version": "1.1.8",
```

This should be a major bump as you removed functionality of the unofficial domains
```suggestion
	"version": "2.0.0",
```